NF-Tools-Schemas/animal-model/submitAnimalModel.json

```
@@ -0,0 +1,190 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Submit Animal Model",
+  "type": "object",
+  "properties": {
+    "animalModelName": {
+      "type": "string",
+      "title": "What is the name of the animal model?"
+    },
+    "synonyms": {
+      "type": "string",
+      "title": "Are there any synonyms?"
+    },
+    "description": {
+      "type": "string",
+      "title": "Please provide a description of the animal model (~20 words)"
+    },
+    "backgroundStrain": {
+      "type": "string",
+      "title": "What is the background strain?"
+    },
+    "backgroundSubstrain": {
+      "type": "string",
+      "title": "What is the background substrain?"
+    },
+    "strainNomenclature": {
+      "type": "string",
+      "title": "What is the strain nomenclature?"
+    },
+    "animalModelDisease": {
+      "type": "string",
+      "title": "What is the animal model disease?",
+      "enum": [
+        "Neurofibromatosis type 1",
+        "Neurofibromatosis type 2",
+        "No known disease",
+        "Schwannomatosis"
+      ]
+    },
+    "animalModelManifestation": {
+      "type": "string",
+      "title": "What is the animal model of manifestation?",
+      "enum": [
+        "astrocytoma",
+        "high grade glioma",
+        "malignant peripheral nerve sheath tumor",
+        "optic nerve glioma",
+        "spinal development",
+        "plexiform neurofibroma",
+        "metabolic function",
+        "heart malformation",
+        "neural crest hyperplasia",
+        "cognition",
+        "growth",
+        "acute myeloid leukemia",
+        "no symptoms",
+        "Other"
+      ]
+    },
+    "mutationTypes": {
+      "type": "array",
+      "title": "Do any of the mutation types apply?",
```

```suggestion
      "title": "Do any of the mutation types apply?",
```

suggest putting gene symbol question first